_maps/wawastation.json

```diff
@@ -3,6 +3,7 @@
 	"map_name": "Wawastation",
 	"map_path": "map_files/wawastation",
 	"map_file": "wawastation.dmm",
+	"exclude_from_ci": false,
```

Comment:
This needs to be false by default--IMO tg should probably never set this to true so we shouldn't treat it as a reasonable option outside of extreme circumstances

Reply:
Essentially your PR should not touch any existing json files